content/docs/1.7.0/nodes-and-volumes/nodes/node-conditions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Node Conditions
+weight: 7
+---
+
+Node conditions are introduced from Longhorn version 1.7.0. They are used to check the environment settings of worker nodes to identify potential issues before any system impact.
+
+Node conditions:
+
+- `Ready`:  
+  Indicates that the node is ready for Longhorn operations, including that a `longhorn-manager` pod is running on this node, the Kubernetes node is ready, and there is no physical resources pressure.  
+
+- `Schedulable`:  
+  Indicated that the node is not cordoned and workload can be scheduled to this node.
+
+- `MountPropagation`:  
+  Indicates that the node supports mount propagation. This is necessary for sharing of volumes mounted by a container with other containers in the same Longhorn pod, or to other Longhorn pods on the same node.  
+
+- `Multipathd`:  
+  Confirms if the `multipathd` service is not running on the node, which may affect the pod with the volume startup. See [Troubleshooting: `MountVolume.SetUp failed for volume` due to multipathd on the node](../../../../../kb/troubleshooting-volume-with-multipath).  
+
+- `RequiredPackages`:  
```

Comment:
Will this condition be updated when the data engine changes? for example, v1/v2 exclusive or both enabled? Or right now, we just design this for v1 only, and will be improved in the future?  

cc @derekbit

Reply:
Yes, we have a [ticket](https://github.com/longhorn/longhorn/issues/7874) for v2 data engine as well. You're right. We need to check the packages and dependencies based on the enabled data engines.

Reply:
Yeah. Just make sure the check respecting the data engine when v2 is ready.